### ISSUE TITLE
Look into getting added to Shields.io

### ISSUE THREAD

**Issue by kavdev** [pyupio/pyup#180]
There are a bunch of services on the homepage of http://shields.io/ that allow you to customize the style of the shield. We like to use the flat-squared style over at dj-stripe; it'd be great to be able to use the pyup badge

**Comment by jayfk** [comment]
Yeah, that'd be awesome. It looks like the project is in deep hibernation though, with over [50 open pull requests](https://github.com/badges/shields/pulls). I'll see what I can do.
